src/Processors/QtiV2/In/Interactions/HottextInteractionMapper.php

```diff
@@ -58,7 +58,7 @@ private function buildTemplate(HottextInteraction $interaction)
 
             // Inject token spans where there are inline hottext nodes
             // HACK: because these tokens are inline, make sure there is space between the token and its inline siblings
-            $content = str_replace($hottextString, ' '.QtiMarshallerUtil::marshall($tokenSpan).' ', $content);
+            $content = str_replace($hottextString, ' '.QtiMarshallerUtil::marshall($tokenSpan).'', $content);
```

Comment:
Could you please remove `.''` and add spaces for `.`?
```
$content = str_replace($hottextString, ' ' . QtiMarshallerUtil::marshall($tokenSpan), $content);
```

Reply:
Done

Reply:
Just checking in on this one @pooja31chaudhary, I think you may have misunderstood the suggestion. Is there a reason for the trailing appended empty string? Can it just be removed? Also I've noticed tab indentation creeping back in, wondering if you can fix that up quickly.

Reply:
Done